test/test-tasks.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+source $(dirname $0)/common.sh
+
+TMPF=$(mktemp /tmp/.mm.XXXXXX)
+clean() { rm -f ${TMPF}; }
+trap clean EXIT
+
+# Configure the number of parallel tests running at the same time, start from 0
+MAX_NUMBERS_OF_PARALLEL_TASKS=2 # => 8
```

Comment:
What does the `start from 0` and `# => 8` mean here?

Reply:
just a leftover, removed it.